client/nimiq-client.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Nimiq's Rust client
+After=syslog.target network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=forking
```

Comment:
For our case, this should probably be changed to `Type=exec` as described [here](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=)